### PR TITLE
Addition of VS2015 solution files and UWP E2E test app

### PR DESCRIPTION
**Pull request by gb92**
Work related to adding a UWP E2E test app Issue #99. This successfully runs all Win8 tests except Windows Live Client SDK tests and the auth tests.

Disabled Code Analysis in order to successfully build the solution in VS2015.

### CODE REVIEW

**Comment by gb92** [comment]
@dhei @fabiocav 

Can you guys give this a review? Thanks

File: unittest/Microsoft.WindowsAzure.MobileServices.SQLiteStore.WindowsStore.Test/Microsoft.WindowsAzure.Mobile.SQLiteStore.Win8.Test.csproj [hunk 1]
```diff
@@ -196,15 +199,18 @@
     <SDKReference Include="Microsoft.VCLibs, version=11.0">
       <Name>Microsoft Visual C++ Runtime Package</Name>
     </SDKReference>
-    <SDKReference Include="SQLite.WinRT, Version=3.11.0">
-      <Name>SQLite for Windows Runtime</Name>
+    <SDKReference Include="Microsoft.VCLibs, version=12.0">
+      <Name>Microsoft Visual C++ 2013 Runtime Package for Windows</Name>
+    </SDKReference>
```
Comment (dhei) [inline unittest/Microsoft.WindowsAzure.MobileServices.SQLiteStore.WindowsStore.Test/Microsoft.WindowsAzure.Mobile.SQLiteStore.Win8.Test.csproj]: What is the usage of this C++ reference, can you explain?

File: e2etest/UWP.E2ETest/App.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+using Windows.ApplicationModel.Activation;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
```
Comment (dhei) [inline e2etest/UWP.E2ETest/App.xaml.cs]: Missing Microsoft copyright header...

File: e2etest/E2ETestFramework/E2ETestFramework.csproj [hunk 3]
```diff
@@ -26,7 +26,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <RunCodeAnalysis>false</RunCodeAnalysis>
```
Comment (fabiocav) [inline e2etest/E2ETestFramework/E2ETestFramework.csproj]: We need to come back and make sure this works, so can you create an issue for this so it doesn't slip?

File: src/Microsoft.WindowsAzure.MobileServices/Microsoft.WindowsAzure.Mobile.csproj [hunk 4]
```diff
@@ -28,7 +28,7 @@
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>
     </DocumentationFile>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <RunCodeAnalysis>false</RunCodeAnalysis>
```
Comment (fabiocav) [inline src/Microsoft.WindowsAzure.MobileServices/Microsoft.WindowsAzure.Mobile.csproj]: Include this when we create an issue.

File: unittest/Microsoft.WindowsAzure.MobileServices.TestFramework/Microsoft.WindowsAzure.Mobile.TestFramework.csproj [hunk 5]
```diff
@@ -25,7 +25,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <RunCodeAnalysis>false</RunCodeAnalysis>
```
Comment (fabiocav) [inline unittest/Microsoft.WindowsAzure.MobileServices.TestFramework/Microsoft.WindowsAzure.Mobile.TestFramework.csproj]: Include this when we create an issue.

**Pull request merged**